assembl/static2/js/app/components/administration/saveButton.jsx

```diff
@@ -0,0 +1,192 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { gql, graphql, withApollo, compose } from 'react-apollo';
+import { Button } from 'react-bootstrap';
+import { Translate, I18n } from 'react-redux-i18n';
+import { displayAlert } from '../../utils/utilityManager';
+
+const GetThematics = gql`
+{
+  thematics(identifier:"survey") {
+    id,
+    titleEntries {
+      localeCode,
+      value
+    },
+    imgUrl,
+    video {
+      title,
+      description,
+      htmlCode
+    }
+    questions {
+      titleEntries {
+        localeCode,
+        value
+      }
+    }
+  }
+}
+`;
+
+const createLanguageEntries = (titles) => {
+  return titles.map((title) => {
+    return { value: title.value, localeCode: title.localeCode };
+  });
+};
+
+const createQuestionEntries = (questions) => {
+  const questionsArray = [];
+  questions.forEach((question) => {
+    const titlesArray = [];
+    question.titleEntries.forEach((title) => {
+      titlesArray.push({
+        value: title.value,
+        localeCode: title.localeCode
+      });
+    });
+    questionsArray.push({ titleEntries: titlesArray });
+  });
+  return questionsArray;
+};
+
+const createVideoEntries = (v) => {
+  const video = {
+    titleEntries: [{ value: v.title, localeCode: 'fr' }],
+    descriptionEntries: [{ value: v.description, localeCode: 'fr' }],
+    htmlCode: v.htmlCode
+  };
+  return video;
+};
+
+const SaveButton = ({ client, createThematic, updateThematic, deleteThematic, thematicsToDelete }) => {
+  const saveAction = () => {
+    const thematicsData = client.readQuery({ query: GetThematics });
```

Comment:
Fixed it and reverted it. We should rather disable the Save button if there is no change on the data